public/content/eth/supply/index.md

```diff
@@ -0,0 +1,42 @@
+---
+title: ETH supply
+description: How Does ETH Supply Work
+lang: en
+---
+
+## How Many ETH Are There?
+Unlike Bitcoin, which has a capped supply of 21 million coins, Ethereum doesn't have a predefined limit on the total number of ETH that can exist. Instead, the supply grows over time through a process called mining, where miners validate transactions and add them to the Ethereum blockchain. In return for their computational work, miners receive newly minted ETH as rewards.
```

Comment:
```suggestion
Unlike Bitcoin, which has a capped supply of 21 million coins, Ethereum doesn't have a predefined limit on the total number of ETH that can exist. Instead, the supply grows over time through a process called mining, where miners validate transactions and add them to the Ethereum blockchain. In return for their computational work, miners receive newly minted ETH as rewards.
```
there is no mining anymore in Ethereum, this would be misleading to open  #the article with this topic. Should be removed.